### DISCUSSION TITLE
More details about opacity configuration

### DISCUSSION THREAD

**Discussion by vanbang9710** [tomasklaen/uosc#784] [Q&A] [answered]
In uosc.conf
```
# A comma delimited list of opacity overrides for various UI element backgrounds and shapes.
# This does not affect any text, which is always rendered fully opaque. Defaults:
# timeline=0.9,position=1,chapters=0.8,slider=0.9,slider_gauge=1,controls=0,speed=0.6,menu=1,submenu=0.4,border=1,title=1,tooltip=1,thumbnail=1,curtain=0.8,idle_indicator=0.8,audio_indicator=0.5,buffering_indicator=0.3,playlist_position=0.8
```
Where to find images of all these elements?

**Comment by tomasklaen** [comment, marked as answer]
- **timeline** - timeline backgorund
- **position** - timeline _position_, i.e. the line or bar depending on configuration
- **chapters** - the diamond chapter indicators at the top of the timeline
- **slider** - slider (volume) background
- **slider_gauge** - slider (volume) value bar
- **controls** - control button backgrounds
- **speed** - speed control backgorund
- **menu** - menu backgorund
- **submenu** - the opacity of a peeked submenu when hovering it's parent item
- **border** - window border
- **title** - window title background
- **tooltip** - any tooltip background
- **thumbnail** - thumbnail border
- **curtain** - the fadeout when menu is open
- ***_indicator** - the icons visible in the middle of the window when there's no video, audio is playing, or video is buffering
- **playlist_position** - the playlist position button next to window title if there's a playlist

Window title and playlist position are only visible in mpv's `no-border` mode, or when forced to be displayed in `uosc.conf`.
